.travis.yml

```diff
@@ -2,4 +2,20 @@ language: javascript
 
 install: "pip install --user $(whoami) -r requirements.txt"
 
-script: ""
\ No newline at end of file
+script: ""
```

Comment:
Maybe this is connected to the problem with the docker image? #4 
virtualenv causes exception during the booting of workers.